custom_components/powercalc/data/govee/H6076/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TP-Link, P110",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 30,
+        "VERSION": "v1.9.8:docker"
+    },
+    "name": "H6076",
```

Comment:
Name can be set to the fully marketed name for the light. i.e. `Govee RGBICW Smart Corner Floor Lamp`